frontend/PreferencesPanel.js

```diff
@@ -90,7 +90,7 @@ class PreferencesPanel extends React.Component {
               ref={this._setSelectRef}
               value={themeName}
             >
-              {!showHiddenThemes && (<option value="">{browserName}</option>)}
+              {browserName && (<option value="">{browserName}</option>)}
```

Comment:
> To be honest I'm still a little unclear on the benefit of this feature (matching theme between standalone devtools and a browser). Am I misunderstanding it?

The context is I made the `match devtools` feature for [`react-native-debugger`](https://github.com/jhen0409/react-native-debugger), see the screenshot:

![2017-06-02 06_01_50](https://cloud.githubusercontent.com/assets/3001525/26702753/0a04b31e-4759-11e7-9f4a-09f90f4aface.gif)

The problem is the `localStorage.themeName` must be empty if we want to tweak theme automatically , but it will never tweak if user selected a theme, so we need a way to back.

So it will not called as `Chrome` in this case, or some case for whos use the standalone devtools with the added method of this PR.